pages/dashboard/github-integration.md

```diff
@@ -0,0 +1,153 @@
+# Integração com o GitHub
+
+O [Cypress Dashboard](https://on.cypress.io/dashboard) consegue integrar seus testes Cypress
+com seu Workflow do GitHub através de [verificações de status](#verificacoes-de-status)
+e comentários de [pull requests](#comentarios-de-pull-requests). 
+
+Um projeto inicialmente precisa estar [configurado para gravação](https://docs.cypress.io/guides/dashboard/projects)
```

Comment:
Precisa adicionar linha TODO acima do texto para atualizar o link quando a página for traduzida.